.github/workflows/autodoc_cicd.yml

```diff
@@ -98,34 +98,34 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ~/.cache/pip
-          key: Python-pydita-generatedcommands-v${{ env.RESET_PIP_CACHE_2 }}-${{ inputs.MAIN_PYTHON_VERSION }}-${{ hashFiles('pyproject.toml') }}
+          key: Python-pyconverter-generatedcommands-v${{ env.RESET_PIP_CACHE_2 }}-${{ inputs.MAIN_PYTHON_VERSION }}-${{ hashFiles('pyproject.toml') }}
           restore-keys: |
-            Python-pydita-generatedcommands-v${{ env.RESET_PIP_CACHE_2 }}-${{ inputs.MAIN_PYTHON_VERSION }}
+            Python-pyconverter-generatedcommands-v${{ env.RESET_PIP_CACHE_2 }}-${{ inputs.MAIN_PYTHON_VERSION }}
       
-      - name: "Install pydita-generatedcommands"
+      - name: "Install pyconverter-generatedcommands"
         run: |
           pip install .
       
-      - name: "Retrieve PyDita-Generatedcommands version"
+      - name: "Retrieve PyConverter-Generatedcommands version"
         run: |
-          echo "PYDITA_GENERATED_VERSION=$(python -c 'from pydita.generatedcommands import __version__; print(__version__)')" >> $GITHUB_ENV
-          echo "PyDita-Generatedcommands version is: $(python -c 'from pydita.generatedcommands import __version__; print(__version__)')"
+          echo "PYCONVERTER_GENERATED_VERSION=$(python -c 'from pyconverter.generatedcommands import __version__; print(__version__)')" >> $GITHUB_ENV
+          echo "PyConverter-XML2Py-Generatedcommands version is: $(python -c 'from pyconverter.generatedcommands import __version__; print(__version__)')"
```

Comment:
I'm curious here... I can't seem to find ``pyconverter.generatedcommands``... Moreover, shouldn't it be ``pyconverter.xml2py.generatedcommands``

Reply:
I named the autogenerated package as `pyconverter-generatedcommands` to be shorter. This can be changed.

Reply:
Ah got it!